src/components/MultiSelect/MultiSelect.test.tsx

```diff
@@ -143,153 +110,134 @@ describe.skip("MultiSelect Dialog", () => {
-        type="dialog"
+        id="multiselect-test-id"
+        helperText="multiselect-helper-text"
+        buttonText="multiselect-button-text"
+        isDefaultOpen={false}
+        isSearchable={false}
+        isBlockElement={false}
+        defaultItemsVisible={defaultItemsVisible}
         items={items}
         selectedItems={selectedTestItems}
         onChange={() => null}
         onClear={() => null}
-        onApply={() => null}
       />
     );
-
-    expect(
-      screen.getByRole("button", {
-        name: /multiselect test label/i,
-      })
-    ).toBeInTheDocument();
+    expect(screen.getByRole("button")).toBeInTheDocument();
   });
 
   it("should initially render with a closed menu when the isDefaultOpen is omitted or set to false", () => {
-    const { rerender } = render(
-      <MultiSelect
-        id="multiselect-dialog-test-id"
-        labelText="MultiSelect Test Label"
-        type="dialog"
-        items={items}
-        selectedItems={selectedTestItems}
-        onChange={() => null}
-        onClear={() => null}
-        onApply={() => null}
-      />
-    );
-
-    expect(screen.queryByRole("checkbox")).toBeNull();
-
-    rerender(
+    render(
       <MultiSelect
-        id="multiselect-dialog-test-id"
-        labelText="MultiSelect Test Label"
-        type="dialog"
-        items={items}
+        id="multiselect-test-id"
+        helperText="multiselect-helper-text"
+        buttonText="multiselect-button-text"
         isDefaultOpen={false}
+        isSearchable={false}
+        isBlockElement={false}
+        defaultItemsVisible={defaultItemsVisible}
+        items={items}
         selectedItems={selectedTestItems}
         onChange={() => null}
         onClear={() => null}
-        onApply={() => null}
       />
     );
-
     expect(screen.queryByRole("checkbox")).toBeNull();
   });
 
   it("should initially render with open menu if isDefaultOpen prop is true", () => {
     render(
       <MultiSelect
-        id="multiselect-dialog-test-id"
-        labelText="MultiSelect Test Label"
-        type="dialog"
-        items={items}
+        id="multiselect-test-id"
+        helperText="multiselect-helper-text"
+        buttonText="multiselect-button-text"
         isDefaultOpen={true}
+        isSearchable={false}
+        isBlockElement={false}
+        defaultItemsVisible={defaultItemsVisible}
+        items={items}
         selectedItems={selectedTestItems}
         onChange={() => null}
         onClear={() => null}
-        onApply={() => null}
       />
     );
-
+    expect(screen.getByRole("button").getAttribute("aria-expanded")).toEqual(
+      "true"
+    );
     expect(screen.getAllByRole("checkbox")).toHaveLength(8);
   });
 
-  it("should render a 'Clear' and 'Apply button", () => {
+  it("should initially render with open menu if isDefaultOpen prop is true and when items are selected", () => {
+    selectedTestItems = { "multiselect-test-id": { items: ["red", "blue"] } };
```

Comment:
For this test, you should verify that the checkboxes that has "red" and "blue" are checked.